readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Dka888.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Dka888.github.io/layout_product-cards/report/html_report/)
 
```

Comment:
it looks like the demo link and the test report link aren't working :(

![image](https://user-images.githubusercontent.com/84134018/224554240-e074aa20-c907-4ef4-8e89-57ef1200f3d8.png)

![image](https://user-images.githubusercontent.com/84134018/224554253-d8299f31-eb91-43e7-b9e9-49ca8914cc4b.png)